src/test/java/us/kbase/test/common/MapBuilder.java

```diff
@@ -1,4 +1,4 @@
-package us.kbase.common.test;
+package us.kbase.test.common;
```

Comment:
OK, so the `test` directory has been moved up a level instead of having a test dir under each of the three subdirectories. 👍